rts/motoko-rts/src/debug.rs

```diff
@@ -165,7 +165,7 @@ pub(crate) unsafe fn print_boxed_object(buf: &mut WriteBuf, p: usize) {
             let obj_ind = obj as *const ObjInd;
             let _ = write!(buf, "<ObjInd field={:#x}>", (*obj_ind).field.get_raw());
         }
-        TAG_ARRAY => {
+        TAG_ARRAY | TAG_ARRAY_SLICE_MIN.. => {
```

Comment:
Thank you very much. This is indeed a missing case of `TAG_ARRAY_SLICE_MIN` during the incremental mark phase.